docs/source/examples/simpleflight.py

```diff
@@ -1,44 +1,43 @@
-import numpy as np
-
+from gpkit.shortcuts import *
 import gpkit
+import numpy as np
```

Comment:
by convention imports should be ordered by standard library stuff first, then site packages. Technically this means it's ok for numpy to come after gpkit, but I think most python developers would expect numpy to come first since it's so "standard".